dashboard/src/components/AppList/AppList.tsx

```diff
@@ -111,7 +103,7 @@ function AppList() {
             />
             {canSetAllNS && (
               <CdsToggleGroup className="flex-v-center">
-                <CdsToggle>
+                <CdsToggle data-testid="cds-all-ns">
```

Comment:
Why this `testid` ?

Reply:
Argh - I'd added it when struggling to get the test to work, but forgot to remove. Thanks.